iroh-rpc-types/proto/test.proto

```diff
@@ -0,0 +1,5 @@
+syntax = "proto3";
+
+package test;
+
+service Test {}
```

Comment:
needed an "empty" service, so I could use a service for testing purposes without having to re-implement each given method.

Could have just used the `gateway` tonic server for now, which has no methods currently, but that will likely change soon.

Wasn't sure if it would be better to pull this definition/build into the `iroh-rpc-client` crate, where it is being used in the client tests.

Reply:
seems fine to have it here for now

Reply:
if you want to make it nicer, you can put it behind a feature like `testing` and then activate that only in the dev dependencies of the client, so it doesn't get used in release builds

Reply:
damn that was so easy to implement!